src/fairseq2/recipes/lm/preference_finetune/recipe.py

```diff
@@ -343,12 +346,18 @@ def load_preference_finetuner(
             "The criterion cannot be initialized. See nested exception for details."
         ) from ex
 
+    # Initialize the data reader.
+    if config.batch_size is not None:
+        batching: StaticBatching | LengthBatching = StaticBatching(config.batch_size)
```

Comment:
is this type annotation to make mypy happy?

Reply:
yeah, is there a nicer way to do it?

Reply:
@cbalioglu is `#type: ignore` preferred here?

Reply:
went with first option because IMO changing the codebase to use the TypeAlias makes things less clear in most cases.